allennlp/modules/attention/additive_attention.py

```diff
@@ -17,11 +17,11 @@ class AdditiveAttention(Attention):
 
     Parameters
     ----------
-    vector_dim : ``int``
+    vector_dim : ``int``, required
```

Comment:
I'm confused because some of the lines contain a space before the `:` and others don't.  Should we have a space here or not?

Reply:
A long time ago, we found that we needed to have a space before and after the `:` in order for things to render properly.  I haven't looked at this in a *long* time, so I don't know if something changed from under us, or if I misunderstood something back then.  I'd like to know the answer, too =).

Reply:
@schmmd @matt-gardner 
I understand your confusion. I was also confused. 
As I wrote earlier, 

> Don't add whitespace between argument name and type

is another solution. This is the sphinx style. To be honest, it's better to remove spaces before a colon in terms of the sphinx and docstring readability. 

However, most docstrings are styled by double backquotes. As long as I know, this style is also rendered correctly because sphinx recognizes backquotes as inline code.

Did you resolve a concern? 
If you want to remove white space, I'm willing to. Let me know your preferences!

Reply:
Ah--you're right.  We're entirely inconsistent in our codebase (I just looked at the output of `git grep -A 5 Parameters`).  Because we're already inconsistent that issue is independent of this PR.  Thanks for taking care of this!